content/blog/nats-dotnet-v2-alpha-release.md

```diff
@@ -0,0 +1,148 @@
++++
+date = "2023-07-13"
+draft = false
+title = "NATS .NET Client v2 Alpha Released with Core NATS Support"
+author = "Ziya Suzen"
+categories = ["Engineering"]
+tags = ["NATS", "Client", "Pre-release", ".NET"]
++++
+
+Starting in 2018 with [.NET Core 2.1](https://devblogs.microsoft.com/dotnet/announcing-net-core-2-1/), the .NET platform began to receive serious performance improvements in every major release. This excited many .NET developers, who have been enjoying faster and more efficient runtimes just by upgrading their project targets with every new .NET release. In 2021, [.NET 6](https://devblogs.microsoft.com/dotnet/announcing-net-6/) became the first LTS release under Microsoft’s new [.NET and .NET Core Support Policy](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core).
+
+The original NATS .NET client, called [NATS.Client v1](https://www.nuget.org/packages/NATS.Client) on NuGet, pre-dates these advancements and Microsoft’s new LTS roadmap - it was originally released in 2015! Backwards compatibility remains a large commitment in NATS.Client v1, which would make it difficult to port in order to take advantage of new APIs.
+
+NATS community members started to take note, and develop client libraries for NATS based on modern .NET APIs. One notable client library that emerged was the [AlterNats](https://github.com/Cysharp/AlterNats) library by Cysharp, which includes a fully asynchronous API, leverages [Span&lt;T&rt;](https://learn.microsoft.com/en-us/archive/msdn-magazine/2018/january/csharp-all-about-span-exploring-a-new-net-mainstay), and supports client-side WebSockets from browsers in [Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor). NATS maintainers and AlterNats maintainers agreed that AlterNats would make a great starting point for NATS.Client v2!
```

Comment:
Should this be `&gt;` in `Span&lt;T&rt;`?

Reply:
Bah, good catch.